plugins/opencode-knowledge/commands/knowledge-doctor.md

```
@@ -1,72 +1,133 @@
+1. **Multi-world.** A system is many worlds; URLs are `mark://<world>/<path>`, not bare paths. Enumerate with `mark_worlds` and audit per world.
+2. **The broker graph store is ephemeral** (per-pod lifetime; resets on broker restart). `mark_backlinks` only returns what *this* session has crawled — so you **must** `mark_graph` to populate it before any backlink/orphan reasoning, every run.
+3. **Cross-world references are existence-checkable** here (they aren't in a single-world soul audit): `mark_worlds` + per-world `mark_lookup` can resolve `mark://<other-world>/<path>`. This un-defers the bare-`mark://` reference check (see Deep checks).
 
-Before the first broker call, record one command-wide deadline five minutes in the future. Check remaining time before every `mark_worlds`, `mark_graph`, `mark_list`, `mark_fetch`, `mark_lookup`, `mark_versions`, and `mark_backlinks` call, including root, supplemental, retry, and deep-check calls. Stop starting calls when the deadline expires; preserve collected results, list skipped work, mark affected worlds incomplete, and suppress conclusions that require missing coverage. OpenCode exposes no per-call timeout or cancellation argument for these MCP tools, so the deadline is advisory for a call already in flight; state this limitation in the report if an in-flight call returns after the deadline.
+## Scope
 
-## Gather
+Resolve the audit scope from `$ARGUMENTS`:
+
+- **blank** → every joined system, all worlds. Read the registry for the joined systems:
+  ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)